### DISCUSSION TITLE
ホームに表示される記事を毎回変えたい

### DISCUSSION THREAD

**Discussion by YossyHal** [increments/qiita-discussions#358] [要望・アイデア・不具合]
Qiitaを開いてホームに表示される記事がだいたいいつも同じなので、もっと別の記事も見たいです。
極端な話、ページをリロードする事におすすめ記事がランダムに表示されたりしても良いと思いました。

**Comment by mkdk09** [comment]
@YossyHal 
ご意見ありがとうございます！
ホームのパフォーマンスの都合上現在の仕様となっております🙏
今回いただいた内容も含めて改善を検討いたします！

**Comment by kenta-watanabe** [comment]
数ヶ月前に投稿された記事が数日以上ホーム一番上に表示されるのは仕様でしょうか？
何度か見た記事が上にずっと表示されているため、とても邪魔です。

**Comment by kimkim0814** [comment]
@YossyHal 
お待たせして申し訳ありません！

ホームに表示される記事について、1つの選出パターンではなく、複数の選出パターンよりランダムに選択されるように変更しました

是非お試しいただけると嬉しいです:blush:

@kenta-watanabe
>数ヶ月前に投稿された記事が数日以上ホーム一番上に表示されるのは仕様でしょうか？

週に何回も同じ記事が表示されないようにするロジックが入っているので、改めて不具合がないか調査いたします。
ご不便おかけしてしまい申し訳ございませんが、よろしくお願いいたします🙇

**Comment by tanaka-asami415** [comment]
@kenta-watanabe
お待たせして申し訳ありません。調査したところ不具合はなさそうでした。

現在は基本的には同じ記事はできるだけ表示されないようにしておりますので、もしまだ同じ記事が何度も表示されているようであれば
個別にお調べしますので、[サポートフォーム](https://support.qiita.com/hc/ja/requests/new) よりアカウントをご連絡いただけますと幸いです🙇
